src/radical/pilot/agent/launch_method/srun.py

```diff
@@ -88,6 +88,15 @@ def can_launch(self, task):
         return True, ''
 
 
+    # -------------------------------------------------------------------------
+    #
+    def get_slurm_ver(self):
+        
+        version = eval(self._version.strip('\n'))
```

Comment:
I don't understand what the purpose of the `eval` is here - can you explain please?

Reply:
@andre-merzky I committed a change. The eval should be on the major version not here! The eval should evaluate the `str` to `int` to be used in the version check later. I replaced it with `int` as it is safer!